### ISSUE TITLE
Description of GroupNormalization vs. LayerNormalization on tensorflow.org/addons is incorrect

### ISSUE THREAD

**Issue by chaene** [tensorflow/addons#1720]
The tutorial

https://www.tensorflow.org/addons/tutorials/layers_normalizations

mentions that the LayerNormalization layer from core tensorflow is equivalent to GroupNormalization from tensorflow addons using groups=1. However this is not the case for CNNs due to the specific implementation that was chosen in the LayerNormalization layer.

The way how to check this is as follows:

```
import tensorflow.compat.v2 as tf
import tensorflow_addons as tfa

tf.compat.v1.enable_eager_execution()

inputs = tf.random.truncated_normal(shape=(2, 3, 4, 5))
group_norm = tfa.layers.GroupNormalization(groups=1)
outputs = group_norm(inputs)
layer_norm = tf.keras.layers.LayerNormalization()
outputs_layer_norm = layer_norm(inputs)
outputs_group_norm = group_norm(inputs)

print('gamma_shape_group_norm', group_norm.gamma.shape)
print('gamma_shape_layer_norm', layer_norm.gamma.shape)

print('difference =',tf.norm(outputs_layer_norm-outputs_group_norm).numpy())


gamma_shape_group_norm (5,)
gamma_shape_layer_norm (5,)
difference = 5.0234647
```

The reason for this is that the LayerNormalization layer only normalized along axis=-1 and the GroupNormalization Layer normalizes along all axes but the batch axis. Changing the axes LayerNormalization normalizes over using axis=[1,2,3] will also not result in the desired equivalence because then the shape of gamma will be [height, width, channels] for the LayerNormalization layer. As far as I can tell there is currently no way to make the LayerNormalization layer do the same computation as the GroupNormalization layer with groups=1.

**Comment by AakashKumarNain** [comment]
Thanks for reporting this. @smokrow can you look into this?
